api/src/exam-environment/routes/exam-environment.ts

```diff
@@ -567,10 +568,17 @@ async function postExamAttemptHandler(
  */
 async function postScreenshotHandler(
   this: FastifyInstance,
-  _req: UpdateReqType<typeof schemas.examEnvironmentPostScreenshot>,
+  req: UpdateReqType<typeof schemas.examEnvironmentPostScreenshot>,
   reply: FastifyReply
 ) {
-  return reply.code(418);
+  void reply.code(200).send();
```

Comment:
Before sending the 200, the body should be validated in some way. Either for length or content type - that it is not corrupt. That way, the client can be let known if anything is wrong with what it sent.

Reply:
As discussed, will be verifying the JPG image using magic numbers